src/subcommands/logs.ts

```diff
@@ -1,20 +1,29 @@
 // Copyright 2021 Deno Land Inc. All rights reserved. MIT license.
 
+import type { Args } from "../args.ts";
 import { wait } from "../../deps.ts";
 import { error } from "../error.ts";
 import { API, APIError } from "../utils/api.ts";
+import type { Project } from "../utils/api_types.ts";
 
 const help = `deployctl logs
-Stream logs for the given project.
+View logs for the given project. It supports both live logs where the logs are streamed to the console as they are
+generated, and query persisted logs where the logs generated in the past are fetched.
 
-To show the latest logs of a project:
+To show the live logs of a project's latest deployment:
   deployctl logs --project=helloworld
 
-To show the logs of a particular deployment:
+To show the live logs of a particular deployment:
   deployctl logs --project=helloworld --deployment=1234567890ab
 
-To show the logs of the production deployment:
-  deployctl logs --project=helloworld --prod
+To show the live, error & info level logs of the production deployment generated in particular regions:
+  deployctl logs --project=helloworld --prod --levels=error,info --regions=region1,region2
+
+To show the logs generated within [2h ago, 30m ago] and containing the word "foo":
```

Comment:
```suggestion
To show the logs generated since last 2 hours until last 30m ago and containing the word "foo":
```

Reply:
Does "since last 2 hours until last 30m ago" in your suggestion sound natural? What about like this?

```
from 2 hours ago to 30 minutes ago
```

Reply:
or ` logs generated within the past two hours, up until 30 minutes ago` anything is fine as long as it is simple to understand and doesn't have custom syntax.